ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -34,33 +46,77 @@
 | -------- | -------- | -------- | -------- |
 {# ethernet interfaces #}
 {# ------- #}
-{%      if ethernet_interfaces is defined and ethernet_interfaces is not none %}
+{%      if ethernet_interfaces is arista.avd.defined %}
 {%          for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
-{%              if ethernet_interfaces[ethernet_interface].isis_enable is defined and ethernet_interfaces[ethernet_interface].isis_enable is not none %}
-| {{  ethernet_interface }} | {{ ethernet_interfaces[ethernet_interface].isis_enable }} |  {{ ethernet_interfaces[ethernet_interface].isis_metric | default('-') }} | {% if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is defined and ethernet_interfaces[ethernet_interface].isis_network_point_to_point is not none %} point-to-point {% elif ethernet_interfaces[ethernet_interface].isis_passive is defined and ethernet_interfaces[ethernet_interface].isis_passive is not none %} passive {% else %} - {% endif %}|
+{%              if ethernet_interfaces[ethernet_interface].isis_enable is arista.avd.defined %}
+| {{  ethernet_interface }} | {{ ethernet_interfaces[ethernet_interface].isis_enable }} |  {{ ethernet_interfaces[ethernet_interface].isis_metric | default('-') }} | {% if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is arista.avd.defined %} point-to-point {% elif ethernet_interfaces[ethernet_interface].isis_passive is arista.avd.defined %} passive {% else %} - {% endif %}|
```

Comment:
Similar issues as commented in other PR: Feat(eos_cli_config_gen): Add loopback node-SID for ISIS-SR

Reply:
Refactored this in 1255 so i just copied the implementation from there.